build.gradle.kts

```diff
@@ -367,6 +367,26 @@ subprojects {
     from(tasks["javadoc"])
   }
 
+  tasks.withType<Jar> {
+    if (name == "sourcesJar") {
+      into("META-INF") {
+        from(rootDir) {
+          include("LICENSE")
+          include("NOTICE")
+        }
+      }
+    } else {
+      into("META-INF") {
+        from(rootDir) {
+          include("LICENSE.bin")
+          rename("LICENSE.bin", "LICENSE")
+          include("NOTICE.bin")
+          rename("NOTICE.bin", "NOTICE")
+        }
+      }
+    }
+  }
+
```

Comment:
It seems that we can shorten the code as follows:
```
   into("META-INF") {
      from(rootDir) {
        if (name == "sourcesJar") {
          include("LICENSE")
          include("NOTICE")
        } else {
          include("LICENSE.bin")
          rename("LICENSE.bin", "LICENSE")
          include("NOTICE.bin")
          rename("NOTICE.bin", "NOTICE")
        }
      }
    }
```

anyway, the current is acceptable to me, just a minor suggestion.